react/features/shared-video/constants.ts

```diff
@@ -28,3 +28,13 @@ export const PLAYBACK_STATUSES = {
     PAUSED: 'pause',
     STOPPED: 'stop'
 };
+
+/**
+ * The domain for youtube URLs.
+ */
+export const YOUTUBE_URL_DOMAIN = 'youtube.com';
```

Comment:
aren't there more domains like youtu.be and m.youtube.com and embed.yotube.com?

Reply:
There are! 

I use this as a placeholder and don't use it to match on it. Basically checking if this string is there then youtube is enabled, if not then it is disabled.